importexport/musicxml/exportxml.cpp

```diff
@@ -3211,6 +3211,11 @@ static void writePitch(XmlWriter& xml, const Note* const note, const bool useDru
                   default:                                             break;
                   }
             }
+      // Override accidental with explicit note tuning
+      auto tun = note->tuning();
+      if (fabs(tun) > 0.0) {
```

Comment:
As MuseScore allows tuning on cents only, and as comparing `float` to `0.0` and using it as a 'bool` isn't a good idea anyway, maybe better use `if (fabs(tun) > 0.01) {`, similar to how it is used in https://github.com/musescore/MuseScore/blob/090b8ca0bd9342c8e0b7fdd4b340018af9da1f5a/mscore/musescore.cpp#L6114

Reply:
Fixed the comparison